logsmanagement/helper.h

```diff
@@ -0,0 +1,330 @@
+        case LOGS_MANAG_DEBUG:
+            fprintf(stream, "LOGS_MANAG_DEBUG: ");
+            vfprintf(stream, format, args);
+            break;
+        case LOGS_MANAG_INFO:
+            sprintf(msg, info_msg_prefix);
+            vsnprintf(&msg[sizeof(info_msg_prefix) - 1], msg_len - (sizeof(info_msg_prefix) - 1), format, args);
+            info("%s", msg);
+            break;
+        case LOGS_MANAG_WARNING:
+            sprintf(msg, warn_msg_prefix);
+            vsnprintf(&msg[sizeof(warn_msg_prefix) - 1], msg_len - (sizeof(warn_msg_prefix) - 1), format, args);
+            info("%s", msg);
+            break;
+        case LOGS_MANAG_ERROR:
+            sprintf(msg, error_msg_prefix);
+            vsnprintf(&msg[sizeof(error_msg_prefix) - 1], msg_len - (sizeof(error_msg_prefix) - 1), format, args);
+            error("%s", msg);
+            break;
+        default:
+            break;
+    }
+
+    va_end(args);
+}
+#endif  // LOGS_MANAG_DEBUG_LEV
+
+typedef enum {
+    STR2XX_SUCCESS = 0,
+    STR2XX_OVERFLOW,
+    STR2XX_UNDERFLOW,
+    STR2XX_INCONVERTIBLE
+} str2xx_errno;
+
+/* Convert string s to int out.
+ * https://stackoverflow.com/questions/7021725/how-to-convert-a-string-to-integer-in-c
+ *
+ * @param[out] out The converted int. Cannot be NULL.
+ * @param[in] s Input string to be converted.
+ *
+ *     The format is the same as strtol,
+ *     except that the following are inconvertible:
+ *     - empty string
+ *     - leading whitespace
+ *     - any trailing characters that are not part of the number
+ *     Cannot be NULL.
+ *
+ * @param[in] base Base to interpret string in. Same range as strtol (2 to 36).
+ * @return Indicates if the operation succeeded, or why it failed.
+ */
+static inline str2xx_errno str2int(int *out, char *s, int base) {
+    char *end;
+    if (unlikely(s[0] == '\0' || isspace(s[0]))){
+        // debug(D_LOGS_MANAG, "str2int error: STR2XX_INCONVERTIBLE 1");
+        // m_assert(0, "str2int error: STR2XX_INCONVERTIBLE");
+        return STR2XX_INCONVERTIBLE;
+    }
+    errno = 0;
+    long l = strtol(s, &end, base);
+    /* Both checks are needed because INT_MAX == LONG_MAX is possible. */
+    if (unlikely(l > INT_MAX || (errno == ERANGE && l == LONG_MAX))){
+        // debug(D_LOGS_MANAG, "str2int error: STR2XX_OVERFLOW");
+        // m_assert(0, "str2int error: STR2XX_OVERFLOW");
+        return STR2XX_OVERFLOW;
+    }
+    if (unlikely(l < INT_MIN || (errno == ERANGE && l == LONG_MIN))){
+        // debug(D_LOGS_MANAG, "str2int error: STR2XX_UNDERFLOW");
+        // m_assert(0, "str2int error: STR2XX_UNDERFLOW");
+        return STR2XX_UNDERFLOW;
+    }
+    if (unlikely(*end != '\0')){
+        // debug(D_LOGS_MANAG, "str2int error: STR2XX_INCONVERTIBLE 2");
+        // m_assert(0, "str2int error: STR2XX_INCONVERTIBLE 2");
+        return STR2XX_INCONVERTIBLE;
+    }
+    *out = l;
+    return STR2XX_SUCCESS;
+}
+
+static inline str2xx_errno str2float(float *out, char *s) {
```

Comment:
if the goal of these functions is really to convert string to fload, instead to be here, they should go for `libnetdata/inline.h`.